requirements.in

```diff
@@ -0,0 +1,10 @@
+ArchivesSnake==0.9.1
+Django==3.2.5
+django-cors-headers==3.7.0
+django-mptt==0.12.0
+djangorestframework==3.12.4
+djangorestframework-jwt==1.11.0
```

Comment:
I should have caught this earlier, but `djangorestframework-jwt` is not being used in this app.

Reply:
thanks for catching, thought I checked that!